app/src/main/java/org/mozilla/fenix/perf/PerfFragment.kt

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.perf
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import androidx.fragment.app.Fragment
+import org.mozilla.fenix.HomeActivity
+
+open class PerfFragment : Fragment() {
+    override fun onGetLayoutInflater(savedInstanceState: Bundle?): LayoutInflater {
+        var fragmentInflater = super.onGetLayoutInflater(savedInstanceState)
+        fragmentInflater = PerformanceInflater(fragmentInflater, this.requireContext())
+                .cloneInContext(this.requireContext())
+        fragmentInflater.factory2 = (this.requireActivity() as HomeActivity).factory
```

Comment:
This could crash if this fragment is used with a different activity – can we use `as?` to avoid that?